src/Schemes/Glueing/Types.jl

```diff
@@ -57,6 +57,8 @@ Concrete instance of an `AbsGlueing` for glueings of affine schemes
     if check
       (domain(f) === codomain(g) &&
       domain(g) ===  codomain(f)) || error("maps can not be isomorphisms")
```

Comment:
Could you give a slightly more specific error message? 
Maybe: "domain/codomain mismatch - maps cannot be mutually inverse"